admin-dev/themes/new-theme/js/components/form/form-notification.ts

```diff
@@ -0,0 +1,14 @@
+
+export function notifyFormErrors(jsonResponse: any): void {
+  Object.keys(jsonResponse.errors).forEach((field: string) => {
+    if (Object.prototype.hasOwnProperty.call(jsonResponse.errors, field)) {
+      const fieldErrors: string[] = jsonResponse.errors[field];
+      const errors: string = fieldErrors.join(' ');
+      $.growl.error({message: `${field}: ${errors}`});
+    }
+  });
+};
+
+export default {
+  notifyFormErrors,
+};
```

Comment:
I would prefer to call this file `helpers.ts` or `utils.ts` and stick to that pattern everywhere we create some helpers in the future

This is pretty how I wanted to split the business logic by proposing https://github.com/PrestaShop/ADR/pull/21

Reply:
(Every helpers/utils folders or files could live in their own context folder, the file could just stay here without any problems)